src/Compilers/Core/Portable/Text/TextChange.cs

```diff
@@ -26,8 +29,7 @@ namespace Microsoft.CodeAnalysis.Text
         /// <summary>
         /// The new text.
         /// </summary>
-        [DataMember(Order = 1)]
-        public string? NewText { get; }
+        public string NewText => _newText ?? "";
```

Comment:
📝 This is a public API behavior change for `default(TextChange)`

Reply:
Consider adding a test to capture behavior

---
In reply to: [516919488](https://github.com/dotnet/roslyn/pull/49156#discussion_r516919488) [](ancestors = 516919488)